apps/makefile.comp

```diff
@@ -1,30 +1,32 @@
 # Main make file. All others will simply change the variables within this one
 
-CFLAGS = -g3 -c
+CFLAGS = -g3
```

Comment:
I'm quite surprised this doesn't break a huge amount of the existing code.  Have you tested other code?